javalib/src/main/scala/java/util/zip/ZipOutputStream.scala

```diff
@@ -123,35 +123,35 @@ class ZipOutputStream(_out: OutputStream, charset: Charset)
   }
 
   override def finish(): Unit = {
-    if (out == null) {
+    if (out == null)
       throw new IOException("Stream is closed")
```

Comment:
The JVM throws `IOException` if the stream is closed/null first? Maybe a test for that is appropriate.

Reply:
Good suggestion.

I confirmed that JVM throws an IOException there, then added the Test you suggested.
Understanding the ultimate futility of matching JVM Exception message text, I changed the
SN text in 8 places. 

Maybe re-running CI with these changes will bring me better luck.  Roll those dice!